src/main/java/de/dhbw/karlsruhe/turniere/services/MatchService.java

```diff
@@ -128,7 +128,7 @@ public void setLivescore(Match match, int scoreTeam1, int scoreTeam2) {
      * @param match The Match to start
      */
     public void startMatch(Match match) {
-        if (match.getTeam1() != null && match.getTeam2() != null && match.getState() == Match.State.NOT_STARTED) {
+        if (match.getTeam1().getName() != null && match.getTeam2().getName() != null && match.getState() == Match.State.NOT_STARTED) {
```

Comment:
I don't really like this, but I don't think there's another option with the current implementation...